### PR TITLE
best-practice: fix namespace-classifier value

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

Change namespace-classifier value from "" to "default"

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/2215
- Other reference link(s):

### CODE REVIEW

**Comment by Win-Man** [comment]
LGTM

**Comment by yikeke** [comment]
PTAL @nolouch

**Comment by nolouch** [comment]
ptal @rleungx

File: reference/best-practices/pd-scheduling.md [hunk 1]
```diff
@@ -258,8 +258,8 @@ Similar to slow scheduling, the speed of region merge is most likely limited by
 
 - A lot of tables have been created and then emptied (including truncated tables). These empty regions cannot be merged if the split table attribute is enabled. You can disable this attribute by adjusting the following parameters:
 
-    - TiKV: set `split-region-on-table` to `false`
-    - PD: set `namespace-classifier` to ""
+    - TiKV: Set `split-region-on-table` to `false`. You cannot modify the parameter dynamically.
+    - PD: Set `key-type` to `"txn"` or `"raw"`. You can modify the parameter dynamically.
```
Comment (rleungx) [inline reference/best-practices/pd-scheduling.md]: 3.0 and 2.1 doesn't have this config

Comment (yikeke) [reply reference/best-practices/pd-scheduling.md]: 待会 cherry-pick 后，我们会对 3.0 和 2.1 的版本根据下面俩 PR 做对齐的改动：
- 3.0: https://github.com/pingcap/docs-cn/pull/2506
- 2.1: https://github.com/pingcap/docs-cn/pull/2505

**Comment by toutdesuite** [comment]
@rleungx @yikeke 我把 needs-cherry-pick-3.0 和 needs-cherry-pick-2.1 labels 去掉，重新提一个 PR 对齐这两个版本的中文文档。当前 PR 只保留对 dev 和 v3.1 的修改，这样可以吗~

**Comment by yikeke** [comment]
> @rleungx @yikeke 我把 needs-cherry-pick-3.0 和 needs-cherry-pick-2.1 labels 去掉，重新提一个 PR 对齐这两个版本的中文文档。当前 PR 只保留对 dev 和 v3.1 的修改，这样可以吗~

两种方案都可以。让 bot cherry-pick 后改就不用手动提俩 PR 了，而且改动比较少，很容易改。

**Comment by toutdesuite** [comment]
@yikeke 好的，那让 bot cherry-pick 后再对齐相应的部分

File: reference/best-practices/pd-scheduling.md (hunk 1 above)
Comment (rleungx) [reply reference/best-practices/pd-scheduling.md]: 没问题

Comment (yikeke) [reply reference/best-practices/pd-scheduling.md]: 没其他问题的话请 approve 下～

**Pull request merged**